src/views/layout.njk

```diff
@@ -0,0 +1,98 @@
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"           import govukBackLink %}
```

Comment:
i would remove the govUk components, we won't know what we need yet

Reply:
you mean all imports?

Reply:
line 3 - 22, we can add them in when we need them otherwise we might be importing things we don't need.